### ISSUE TITLE
Parser for new data source: FIRE

### ISSUE THREAD

**Issue by kevinxin90** [biothings/myvariant.info#36]
https://sites.google.com/site/fireregulatoryvariation/

**Comment by sirloon** [comment]
see https://github.com/AnnaTSW0609/FIRE_parser/blob/master/manifest.json for a starter

**Comment by kevinxin90** [comment]
@sirloon If I remember correctly, we once had a discussion with @newgene earlier. And we thought it would be better to first integrate this resource in 'dbnsfp', then we will update the dbnsfp parser to include that.

**Comment by sirloon** [comment]
@kevinxin90 yes, it's just there for reference (or if you want data before dbnsfp is updated - do we have an update from them?)

**Comment by kevinxin90** [comment]
I see. Based on their [update history](https://sites.google.com/site/jpopgen/dbNSFP), it seems there hasn't been any new resources integrated in dbNSFP for over a year now.
